test/regression/cases/otlp_ingest_metrics/experiment.yaml

```diff
@@ -4,11 +4,20 @@ erratic: false
 target:
   name: datadog-agent
   cpu_allotment: 4
-  memory_allotment: 2GiB
+  memory_allotment: 7.5 GiB
 
   environment:
     DD_API_KEY: a0000001
     DD_HOSTNAME: smp-regression
+    # Enable the built-in OTLP receiver
+    DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT: 0.0.0.0:4317
```

Comment:
In general, prefer to put specific config options in `datadog.yaml`, lets try to only use env var if its required (as is the case for api key and hostname)

Reply:
Agreed. I could not figure out how to configure this in `datadog.yaml`.